manifests/l/LingitAS/Lingdys4/4.5.111.0/LingitAS.Lingdys4.locale.nb-NO.yaml

```diff
@@ -0,0 +1,29 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: LingitAS.Lingdys4
+PackageVersion: 4.5.111.0
+PackageLocale: nb-NO
+Publisher: Lingit AS
+PublisherUrl: https://lingit.no/
+PublisherSupportUrl: https://lingit.no/
+PrivacyUrl: https://lingit.no/
+Author: LingIT
+PackageName: Lingdys 4
+PackageUrl: https://lingit.no/
+License: None

```

Comment:
A URL isn't a license; Please update this with actual license terms such as Proprietary, MIT, GPL-2.0, or other terms as appropriate